### PR TITLE
fix: use backtick to format static strings used in apparmor profiles

### PR DESCRIPTION
**Pull request by Revolyssup**
**Purpose of PR?**:

Excessive use of string concatenation with `+` operator for static strings makes the code hard to read. This PR replaces it with a single multi-line string created using backtick

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Commit has unit tests
- [ ] Commit has integration tests

### CODE REVIEW

File: KubeArmor/enforcer/appArmorEnforcer.go [hunk 1]
```diff
@@ -340,28 +339,27 @@ func (ae *AppArmorEnforcer) CreateAppArmorHostProfile() error {
...
 
+file,
+network,
+capability,
+## == PRE END == ##
+
+## == POLICY START == ##
+## == POLICY END == ##
+}
+	`
```
Comment (kranurag7) [inline KubeArmor/enforcer/appArmorEnforcer.go]: Please remove the extra spaces from this line and in the other one too.

**Review by kranurag7 — CHANGES_REQUESTED** [review]
Hey @Revolyssup Thanks for raising the PR. I like the concept of using raw strings. Please see some minor inline comments. 
Docker also [uses](https://github.com/moby/moby/blob/master/profiles/apparmor/template.go) raw strings.

**Comment by nyrahul** [comment]
This definitely makes the code more readable. Thanks @Revolyssup

**Review by daemon1024 — APPROVED** [review]

File: KubeArmor/enforcer/appArmorEnforcer.go (hunk 1 above)
Comment (daemon1024) [inline KubeArmor/enforcer/appArmorEnforcer.go]: @Revolyssup Can we handle this? as @kranurag7 pointed out?

```suggestion
}
`
```

**Review by daemon1024 — CHANGES_REQUESTED** [review]

File: KubeArmor/enforcer/appArmorEnforcer.go (hunk 1 above)
Comment (Revolyssup) [reply KubeArmor/enforcer/appArmorEnforcer.go]: @daemon1024 Sorry for the delay. Yeah I will make the changes today. Thanks for the reminder.

**Comment by Revolyssup** [comment]
@daemon1024 @kranurag7 Please approve CI. I removed redundant spaces.

**Review by daemon1024 — APPROVED** [review]
LGTM, The CI failure was fixed recently. Can you please rebase your PR? Apologies for this taking so long to merge.

**Comment by Revolyssup** [comment]
@daemon1024 pls approve ci

**Review by kranurag7 — APPROVED** [review]
LGTM, Thanks @Revolyssup

**Pull request merged**